completions/VBoxManage

```diff
@@ -0,0 +1,1531 @@
+            createmedium)
+                _vboxmanage_createmedium
+                ;;
+            modifymedium)
+                _vboxmanage_modifymedium
+                ;;
+            clonemedium)
+                _vboxmanage_clonemedium
+                ;;
+            mediumproperty)
+                _vboxmanage_mediumproperty
+                ;;
+            encryptmedium)
+                _vboxmanage_encryptmedium
+                ;;
+            checkmediumpwd)
+                _vboxmanage_checkmediumpwd
+                ;;
+            convertfromraw)
+                _vboxmanage_convertfromraw
+                ;;
+            getextradata)
+                _vboxmanage_getextradata
+                ;;
+            setextradata)
+                _vboxmanage_setextradata
+                ;;
+            setproperty)
+                _vboxmanage_setproperty
+                ;;
+            usbfilter)
+                _vboxmanage_usbfilter
+                ;;
+            sharedfolder)
+                _vboxmanage_sharedfolder
+                ;;
+            guestproperty)
+                _vboxmanage_guestproperty
+                ;;
+            guestcontrol)
+                _vboxmanage_guestcontrol
+                ;;
+            metrics)
+                _vboxmanage_metrics
+                ;;
+            natnetwork)
+                _vboxmanage_natnetwork
+                ;;
+            hostonlyif)
+                _vboxmanage_hostonlyif
+                ;;
+            dhcpserver)
+                _vboxmanage_dhcpserver
+                ;;
+            usbdevsource)
+                _vboxmanage_usbdevsource
+                ;;
+            debugvm)
+                _vboxmanage_debugvm
+                ;;
+            extpack)
+                _vboxmanage_extpack
+                ;;
+            unattended)
+                _vboxmanage_unattended
+                ;;
+        esac
+    fi
+
+} &&
+complete -F _vboxmanage VBoxManage
+
+_vboxmanage_list()
+{
+    if [[ "$cur" == -* ]]; then
+        COMPREPLY=( $( compgen -W "--long --sorted" "$cur" ) )
+    else
+        COMPREPLY=( $( compgen -W "vms runningvms ostypes hostdvds \
+            hostfloppies intnets bridgedifs hostonlyifs natnets \
+            dhcpservers hostinfo hostcpuids hddbackends hdds dvds \
+            floppies usbhost usbfilters systemproperties extpacks \
+            groups webcams screenshotformats" -- "$cur" ) )
+    fi
+}
+
+_vboxmanage_showvminfo()
+{
+    if [[ "$cur" == -* ]]; then
+        COMPREPLY=( $( compgen -W "--details --machinereadable --log" -- \
+            "$cur" ) )
+    else
+        _vboxmanage_get_uuid_vmname
+    fi
+}
+
+_vboxmanage_unregistervm()
+{
+    if [[ "$cur" == -* ]]; then
+        COMPREPLY=( $( compgen -W "--delete" -- "$cur" ) )
+    else
+        _vboxmanage_get_uuid_vmname
+    fi
+}
+
+_vboxmanage_createvm()
+{
+    case $prev in
+        --name)
+            _vboxmanage_get_vmname
+            return 0
```

Comment:
Please replace these with plain returns (no 0) or consistency with rest of the tree.